classes/Mail.php

```diff
@@ -410,10 +410,10 @@ function ($carry, $item) {
             $moduleName = false;
 
             // get templatePath
-            if (preg_match('#' . $shop->physical_uri . 'modules/#', str_replace(DIRECTORY_SEPARATOR, '/', $templatePath)) &&
-                preg_match('#modules/([a-z0-9_-]+)/#ui', str_replace(DIRECTORY_SEPARATOR, '/', $templatePath), $res)
```

Comment:
This function is crazy 🤯 

ping @kpodemski : any idea about why such a complex thing was done to find module template path? 😅

Reply:
🆙 @kpodemski or maybe @PrestaEdit can help me?

Reply:
I'm not sure why it was done like that 🤷 but @tameroski change makes sense